openwisp_notifications/handlers.py

```diff
@@ -394,12 +394,20 @@ def invalidate_cache():
     transaction.on_commit(invalidate_cache)
 
 
+class NotificationUtils:
+    @classmethod
+    def email_notifications_enabled(cls, user):
+        return NotificationSetting.objects.filter(user=user, email=True).exists()
```

Comment:
Please move this method to the abstract `NotificationSetting` model, there's no point in having this class here on its own.

Add a short docstring to the method, eg:

```python
"""Returns ``True`` if ``user`` has at least one email notification setting enabled.""""
```